src/main/java/games/strategy/engine/framework/startup/mc/GameSelectorModel.java

```diff
@@ -283,10 +283,10 @@ private GameChooserEntry selectByName(final boolean forceFactoryDefault) {
     final String userPreferredDefaultGameName = ClientSetting.DEFAULT_GAME_NAME_PREF.value();
 
     final GameChooserModel model = new GameChooserModel();
-    GameChooserEntry selectedGame = model.findByName(userPreferredDefaultGameName);
-    if (selectedGame == null) {
-      selectedGame = model.findByName(userPreferredDefaultGameName);
-    }
+    GameChooserEntry selectedGame = model.findByName(userPreferredDefaultGameName)
+        .orElse(model.findByName(userPreferredDefaultGameName)
+            .orElse(null));
```

Comment:
I may be looking at this wrong, but it appears both the old and new code is calling `model.findByName(userPreferredDefaultGameName)` twice.  Can this just be simplified to the following:

```java
GameChooserEntry selectedGame = model.findByName(userPreferredDefaultGameName).orElse(null);
```

Reply:
It is the same thing twice! 0f72cdf14 removes the redundant call. I tried to track down the history, hard to tell if this was a simplification or an error.